assets/promptflow/.cspell.json

```diff
@@ -66,7 +68,9 @@
       "zelin",
       "embeded",
       "metad",
-      "orignal"
+      "orignal",
```

Comment:
Is this supposed to be "original"?

Reply:
Probably not. "original" would likely already be in the standard dictionary that the spellchecker uses. I suspect there's some resource or sample in this repo with a typo, and instead of correcting the typo, "orignal" got added to the dict here.

It's not a change that I made, and I don't understand enough about why it's here to change it, unfortunately.